activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -471,6 +495,34 @@ def copy_table_contents(from, to, columns, rename = {})
                      SELECT #{quoted_from_columns} FROM #{quote_table_name(from)}")
         end
 
+        def add_column_for_alter(definition, column_name, type, options)
+          definition.column(column_name, type, options)
```

Comment:
Can we make this more easily unified with the other adapters by following the return-a-proc interface, instead of having these perform actions directly?